engine/src/main/java/org/terasology/engine/core/subsystem/common/ConfigurationSubsystem.java

```diff
@@ -66,7 +66,7 @@ public void preInitialise(Context rootContext) {
         checkServerIdentity();
 
         // TODO: Move to display subsystem
-        logger.info("Video Settings: {}", config.renderConfigAsJson(config.getRendering()));
+        logger.atInfo().addArgument(() -> config.renderConfigAsJson(config.getRendering())).log("Video Settings: {}");
```

Comment:
Only evaluated once before initialization.
```suggestion
        logger.info("Video Settings: {}", config.renderConfigAsJson(config.getRendering())); //NOPMD
```